tony-history-server/build.gradle

```diff
@@ -0,0 +1,67 @@
+apply plugin: 'java'
+apply plugin: 'com.google.protobuf'
+
+dependencies {
+  compile deps.external.jackson_databind
+  compile deps.external.py4j
+  compile deps.external.text
+  compile deps.hadoop.common
+  compile deps.hadoop.hdfs
+  compile deps.hadoop.yarn_api
+  compile deps.hadoop.yarn_client
+  compile deps.hadoop.yarn_common
+  compile deps.external.zip4j
+  compile deps.hadoop.mapreduce_client_core
+  compile deps.hadoop.mapreduce_client_hs
+  compile deps.hadoop.mapreduce_client_common
+  compile deps.hadoop.mapreduce_client_app
+  testCompile deps.external.avro
+  testCompile deps.external.testng
+
+  // Only needed by Hadoop test classes
+  testRuntime deps.external.junit
+}
+
+task setupHdfsLib(type: Copy) {
```

Comment:
This task doesn't seem necessary in the tony-history-server module. Looks like leftover from copying the tony-core build.gradle file over.